sdk/servicebus/service-bus/package.json

```diff
@@ -99,7 +99,7 @@
     "@azure/abort-controller": "^1.0.0",
     "@azure/core-amqp": "2.0.0-beta.1",
```

Comment:
Lets also update core-amqp to "2.0.0" here and in Event Hubs

Reply:
done